blockchain.js

```diff
@@ -189,6 +190,45 @@ async function getPastEvents({
   }
 }
 
+async function getEventsRated({
+  chainName,
+  contractName,
+  eventName = 'allEvents',

```

Comment:
Getting all events is bad.

Reply:
This is only the default parameter, it's already overridden in the function calls. It's just a graceful fallback, if you would prefer throwing an error on a missing eventName that's an option.

Reply:
> if you would prefer throwing an error on a missing eventName that's an option.
Yeah I do think that is better, regardless of whether I originally wrote this code or not :P.